app/javascript/components/top-navbar/user-options.jsx

```diff
@@ -0,0 +1,148 @@
+/* global miqToggleUserOptions miqCheckForChanges */
```

Comment:
I think these setting should be in the `/app/javascript/.eslintrc` to set these rules globally. I can imagine that more files would have similar issues.